supply_energy.md

```diff
@@ -0,0 +1,291 @@
+---
+layout: index
+title: Supply of Energy
+prev: inputs_supply.html
+next: outputs_quantity.html
+gcam-version: v5.3 
+---
+
+# Table of Contents
+
+- [Inputs to the Model](#inputs-to-the-model)
+- [Description](#description)
+- [Equations](#equations)
+- [Insights and intuition](#insights-and-intuition)
+- [Policy options](#policy-options)
+- [IAMC Reference Card](#iamc-reference-card)
+- [References](#references)
+
+## Inputs to the Model
+**Table 1: Inputs required by the supply model <sup>[1](#table_footnote)</sup>**
+
+| Name | Resolution | Unit | Source |
+| :--- | :--- | :--- | :--- |
+|  |  |  | [Exogenous](inputs_supply.html) |
+
+
+## Description
+
+### Resources
+
+#### Depletable Resources 
+
+GCAM models depletable resources (oil, unconventional oil, natural gas, coal, and uranium) using graded resource supply curves. The fossil resources are produced from these supply curves using a "Resource / Reserve" model.  In this approach as the market price of the resource increases, we look up the [supply curve](details_energy.html#depletable-resources) to determine the additional quantity available and move that quantity of "resource" into a "reserve".  We assume production of that reserve over some well / mine lifetime appropriate for each fuel.  Technical change can be applied to reduce the extraction cost of the "resource" in future years. Resource curves for fossil fuels are shown below.
```

Comment:
- The [supply curve](details_energy.html#depletable-resources) link does not appear to be working (at least not on the GH render).
- I can't find where the "Resource curves for fossil fuels are shown below."
- Do we want to say anything about the new fossil trade structure?  I'm not sure if this is the most appropriate place for it.  Let me know if it is and you want me to write something up.

Reply:
OK, I see we talk about energy trade below.  It's fine to either link there from here, or simply ignore my second bullet.